docs/blog/newsletter-april-2021.rst

```diff
@@ -0,0 +1,104 @@
+
+.. post:: April 01, 2021
+   :tags: newsletter
+
+######################################
+Write the Docs Newsletter – April 2021
+######################################
+
+Hello everybody! It's Beth and the team, here with the Write the Docs newsletter for April.
+
+Hopefully you'll be pleased to hear we've resisted any tempation to include an April Fool's prank. But I wouldn't want you to think that we're no fun, so here is `a delightful poem about the stuck ship in the Suez canal <https://twitter.com/krfabian/status/1374798847650709504>`__ by a friend of mine.
+
+No big announcements for you this month, but things are ticking along nicely. Portland's coming up in a few weeks, and tickets `are still available </conf/portland/2021/tickets/>`__. If you're undecided, check out the `full schedule </conf/portland/2021/news/announcing-schedule/>`__.
```

Comment:
Is it about time to say we'll stop selling tickets a week (few days?) before the conf?

Reply:
didn't know that - added.